camera_matrix:
  cols: 3
  data:
  - 552.3676403778572
  - 0.0
  - 271.5405270219728
  - 0.0
  - 549.8865688457811
  - 248.41840056895452
  - 0.0
  - 0.0
  - 1.0
  rows: 3

distortion_coefficients:
  cols: 5
  data:
  - -0.1545960462219134
  - 0.008160371579626012
  - 0.002019177429942932
  - 0.002507781319119664
  - 0.1994557667554336
  rows: 1

distortion_model: plumb_bob

image_height: 480
image_width: 640

rectification_matrix:
  cols: 3
  data:
  - 1.0
  - 0.0
  - 0.0
  - 0.0
  - 1.0
  - 0.0
  - 0.0
  - 0.0
  - 1.0
  rows: 3

projection_matrix:
  cols: 4
  data:
  - 552.3676403778572
  - 0.0
  - 271.5405270219728
  - 0.0
  - 0.0
  - 549.8865688457811
  - 248.41840056895452
  - 0.0
  - 0.0
  - 0.0
  - 1.0
  - 0.0
  rows: 3
